mantle/kola/harness.go

```diff
@@ -1564,7 +1568,6 @@ func makeNonExclusiveTest(bucket int, tests []*register.Test, flight platform.Fl
 	if err != nil {
 		plog.Fatalf("Error merging configs: %v", err)
 	}
-
```

Comment:
Can we keep this blank line as it's unrelated to the change?

Reply:
still showing a change here (space added) so the `gofmt` test is complaining. Can you remove the spaces so there is no diff here?

Reply:
should be good to go now, it was just two extra spaces hiding